src/xrpld/app/misc/detail/Manifest.cpp

```diff
@@ -495,6 +495,10 @@ ManifestCache::applyManifest(Manifest m)
 
         auto masterKey = m.masterKey;
         map_.emplace(std::move(masterKey), std::move(m));
+
+        // Something has changed. Keep track of it.
+        seq_++;
+
```

Comment:
From: https://github.com/Xahau/xahaud/pull/631

```
OverlayImpl::getManifestsMessage() caches manifest messages and only rebuilds
them when ManifestCache::sequence() changes. When accepting a new manifest,
the function returned early without incrementing seq_, causing the cache to
never invalidate. This meant peers were exchanging stale lists.

Now seq_++ is called for both new manifests and updates, ensuring the overlay
layer detects changes and sends complete validator lists to connecting peers.
```


I think this has been masked all these years by validator list (e.g. https://vl.ripple.com/) usage which bootstraps manifests